detections/endpoint/winword_spawning_powershell.yml

```diff
@@ -52,3 +52,24 @@ tags:
   - user
   - parent_process_id
   security_domain: endpoint
+  impact: 5000
+  confidence: 1
+  # (impact * confidence)/100
+  risk_score: 50
+  context:
+  - Source:Endpoint
+  - Stage:Execution
+  message: $parent_process_name$ on $dest$ by $user$ launched the following powershell process: $process_name$ which is very common in phishing attacks.
```

Comment:
Hey @apger : looks like our validation is failing on the `value`of the message, can you change `process:` to `process-`

Reply:
alternatively tie that around quotes so it gets treated as a string :smile:

Reply:
Fixed with Bhavin